src/libs/actions/IOU.ts

```diff
@@ -2719,6 +2721,7 @@ function splitBillAndOpenReport(
         reportActionID: splitData.reportActionID,
         createdReportActionID: splitData.createdReportActionID,
         policyID: splitData.policyID,
+        chatType: participants.length > 1 ? CONST.REPORT.CHAT_TYPE.GROUP : '',
```

Comment:
Thought about it some more.. once we fix the optimistic report issue. We can just pass `splitChatReport.chatType` here. Seems better cos otherwise we might pass `chatType: 'group'` for an existing chat that is not a `'group'`.